### ISSUE TITLE
Voting on iPhones does not work

### ISSUE THREAD

**Issue by albi** [lesterchan/wp-polls#75]
Hi there,

got a poll running. Everything is working fine, but not on iPhones. If I try to vote there, nothing is counted, the page reloads and the poll is shown again (logging is set to username).

Had a look at the action that is performed when the form is submitted. It says “/index.php”, but my homepage template is set to page-home.php. Maybe this could be the problem, but why is it working on desktop browsers (Safari, Chrome, Firefox etc.).

Kind regards
Alex

**Comment by lesterchan** [comment]
I am not sure, there can be many things Themes, plugins conflict. I don't think you will get any answer here. Maybe you can try posting on the WP Support Forums.

Check your site for JS errors. If it does not work on one browser, but work on the rest, most likely it is an JS issue.

If that is done, I would suggest changing to the default theme and deactivate all other plugins to see if it works in the first place.

Good luck.

**Comment by albi** [comment]
Lester you were right! I've setup a test scenario with a fresh Wordpress without any other plugins. Since I did not have any Javascript errors on the iPhone with the poll that was not working. After setting up a fresh Wordpress install, I've installed WP-Polls and made the same settings one by one like they are set in the problem-poll. That led me to the templates part of WP-Polls. Some tests later I've seen that my submit button was a normal input type="submit" instead of type="button". I did not know that this could be a problem, because the onclick was the same. But for now I've reverted the type of the input field to button and everything is working as expected.
